megamek/src/megamek/client/ui/swing/MovementDisplay.java

```diff
@@ -4569,7 +4576,7 @@ public void computeMovementEnvelope(Entity suggestion) {
      * @return - This method will do nothing if the Entity passed in is null or
      *           is not an Aero based unity.
      */
-    private void computeAeroMovementEnvelope(Entity entity) {
+    public void computeAeroMovementEnvelope(Entity entity) {
         if ((entity == null) || !(entity instanceof Aero) || (cmd == null)) {
```

Comment:
To handle LAMs correctly this line should also change to entity.isAero(), and a few lines below this should case to IAero instead of Aero.

Reply:
okay - I may need to go back and check to see if I've used `instanceof Aero` in other places as well.